bentoml/service/inference_api.py

```diff
@@ -50,10 +50,12 @@ def __init__(
         mb_max_latency=10000,
         mb_max_batch_size=1000,
         batch=False,
+        route=None,
     ):
         """
         :param service: ref to service containing this API
         :param name: API name
+        :param route: API path (by default the same as `name`)
```

Comment:
Suggest update: 
```
:param route: Specify HTTP URL route of this inference API. By default, `api.name` is used as the route. This parameter can be used for customizing the URL route, e.g. `route="/api/v2/model_a/predict"`
```

Reply:
And should we move the docstring order to match the order of the actual parameter list in this method?

Reply:
done